pkg/app/web/src/components/application-state-view.tsx

```diff
@@ -23,6 +27,7 @@ interface Props {
 }
 
 const ERROR_MESSAGE = "It was unable to fetch the latest state of application.";
+const DISABLED_APPLICATION_MESSAGE = "This application is disabled.";
```

Comment:
"This application is currently disabled. You can enable it from the application list page."

Reply:
I updated the message.
Thank you!

![image](https://user-images.githubusercontent.com/6136383/103988967-49ea3580-51d2-11eb-9513-d3085011d264.png)